rust/src/importer.rs

```diff
@@ -392,7 +392,7 @@ fn path_is_ostree_compliant(path: &str) -> bool {
         return true;
     }
 
-    if path.starts_with("/usr/") && !path.starts_with("/usr/local") {
+    if path.starts_with("/usr/") {
```

Comment:
If we want to be strict, yes. I didn't bother with this because `/usr/local` RPMs are known broken right now anyway and this PR is working to improve the situation. With this hunk but without state overlays enabled, it'll still be broken, but in a different way (though true, in a more subtle way than the current explicit error message users get -- content in `/usr/local` will just be deleted in postprocessing when replaced by the `/var/usrlocal` symlink).

Can pipe through the needed flag to retain current behaviour if wanted.